doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2913,6 +2913,11 @@ You can reset the replication relationship between the global copy and remote co
 
 {product-title} supports database backups to *Network File System (NFS)*, *Samba*, *Amazon Web Service(AWS) S3* and *OpenStack Swift* storage. 
 
+This procedure backs up the following directories:
```

Comment:
These are actually all files, not directories. And also this process here only backs up the database, not files from the filesystem. The user would need to keep these somewhere else to ensure they can be restored.